framework/db/oci/Schema.php

```diff
@@ -426,7 +426,7 @@ protected function createColumn($column)
                         ) {
                             $defaultValue = substr($defaultValue, 1, -1);
                         } else {
-                            $defaultValue = trim($defaultValue);
+                            $defaultValue = trim((string) $defaultValue);
```

Comment:
Wouldn't be better to cast string in line 417? strlen and strncmp and substr all require string to be used, not only trim.

Reply:
Thanks, that should be much better than my choice. I will update this shortly.